packages/weblets/src/elements/disclaimer/Disclaimer.wc.svelte

```diff
@@ -2,15 +2,43 @@
 
 <script lang="ts">
   const envs = {
-    dev: { env: "development", url: "https://github.com/threefoldtech/test_feedback/issues" },
-    test: { env: "testing", url: "https://github.com/threefoldtech/test_feedback/issues" },
-    qa: { env: "QA", url: "https://github.com/threefoldtech/test_feedback/issues" },
-    custom: { env: "customized", url: "https://github.com/threefoldtech/test_feedback/issues" },
+    dev: {
+      env: "development",
+      url: "https://github.com/threefoldtech/test_feedback/issues",
+      playground: "https://playground.dev.grid.tf/#/",
+    },
+    test: {
+      env: "testing",
+      url: "https://github.com/threefoldtech/test_feedback/issues",
+      playground: "https://playground.test.grid.tf/#/",
+    },
+    qa: {
+      env: "QA",
+      url: "https://github.com/threefoldtech/test_feedback/issues",
+      playground: "https://playground.qa.grid.tf/#/",
+    },
+    custom: {
+      env: "customized",
+      url: "https://github.com/threefoldtech/test_feedback/issues",
+      playground: "https://playground.grid.tf/#/",
```

Comment:
why are we using the mainnet's url in the custom one?
I think mainnet should have object as well

Reply:
what is the new playground url for custom network?

Reply:
no need to add it through the config, the old playground is already deprecated